app/locales/en.json

```diff
@@ -100,23 +100,34 @@
     "launch_set_up_phone": "Set up my phone",
     "legal_page_title": "Legal",
     "loading_public_data": "loading data...",
-    "location_disabled_message": "COVID Safe Paths requires location services.",
-    "location_disabled_title": "Location Tracking Was Disabled",
-    "location_enabled_message": "COVID Safe Paths is securely storing your GPS coordinates once every five minutes on this device.",
-    "location_enabled_title": "COVID Safe Paths Enabled",
+    "location_disabled_message": "Please enable location access for Safe Paths.",
+    "location_disabled_title": "Device location disabled",
+    "location_enabled_message": "Safe Paths is anonymously recording your location.",
```

Comment:
@tstirrat  thoughts on this language string change?

Reply:
We iterated on the language stuff in #574, though the diff there makes it hard to see. I think that this is a newer version of that change, is that right @aitkeni ?

The updated language seems fine to me.

Reply:
yes that is correct.  There is a new change which is the word "securely" is changed to "anonymously" in the string "Safe Paths is anonymously recording your location."  This I felt was more accurate and better PR for the app.

Reply:
OK, will close the other one